index.html

```diff
@@ -1,21 +1,219 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <!-- Add a link to your custom CSS here -->
-
+    <link rel="stylesheet" href="css/main.css">
     <title>Responsive grid project</title>
 </head>
+<!-- Add your markup here -->
+<!-- Please leave this in to credit the creators of this design -->
+
 
 <body>
 
-    <!-- Add your markup here -->
 
+    <header class="header-container">
+
+        <div class="search-input">
+            <img src="./plant img/search.png" alt="search" class="search" />
+            <input type="text" id="search" class="search" name="search" placeholder="Search..." />
+
+        </div>
+
+        <div class="login">
+            <div>
+                <img src="./icon-item/login-icon.svg" alt="log in" class="log-in" />
+            </div>
+            <p>Log in</p>
+        </div>
+        <div class="shopping-cart">
+            <img src="./plant img/shopping.png" alt="shopping" class="shopping" />
+        </div>
+        <div class="hamburger"></div>
+        <img src="./plant img/hamburger.png" alt="hamburger" class="hamburger">
+
+
+
+        <nav>
+
+            <ul class="nav-list">
+                <li> <a href="#">Shop</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">FQA</a></li>
+                <li><a href="#">Contact</a></li>
+
+            </ul>
+        </nav>
+        <div class="header-text">
+
+            <h1 class="website-logo">PRICKLES & CO.</h1>
+            
+                <p class="contuct-number">Call Us Now! 123 - 456 -7890</p>
+
+        </div>
+
+    </header>
+    <main class="main">
+        <article class="hero">
+        <section class="welcome">
+
+            <img src="./plant img/background-img.jpeg" alt="background-img" class="background-img">
+
+            <h3 class="esbt">E S B T</h3>
+            <div class="welcome-logo">
+            <img src="./plant img/first-icon.png" alt="first-icon" class="first-icon">
+            </div>
```

Comment:
Please improve your indentation here, it makes it hard to read HTML if it is not properly indented.